File: HTD Training programs/BackEnd/Core Java/objectclass/Student.java

public class Student {
	int id;
	String name;
	public Student(int id, String name) {
		super();
		this.id = id;
		this.name = name;
	}
public String toString() {
	return "student id is : " +id+" student name is :" +name;
}
public static void main(String[] args) {
	Student s=new Student(18,"manju");
	System.out.println(s.toString());
}
}

File: HTD Training programs/FrontEnd/Typescript/student.ts
class Student1 {
    constructor(
        public name: string,
        public age: number,
        public USN: string,
        public marks: number,
        public aadharNo?: number,
        public panCard?:string
    ){

    }
}
let student3=new Student1('name',21,'abc123',80,undefined,'ghf567');
let student2:Student1={
    name: 'pranitha',
    age:20,
    USN: 'dfgh56',
    marks: 69
}
let clearedStudents=[
    new Student1('manju',21,'15ka1a0518',78,989122776321,'Py632')
];
